pcsx2/VMManager.cpp

```diff
@@ -133,6 +133,8 @@ static u32 s_active_no_interlacing_patches = 0;
 static s32 s_current_save_slot = 1;
 static u32 s_frame_advance_count = 0;
 static u32 s_mxcsr_saved;
+static LimiterModeType s_limiter_mode_prior_to_hold_interaction;
```

Comment:
Could probably collapse this to a `std::optional`. And it should be reset on shutdown/initialize.